RM_SUI/RM_SUI/StoreTabBar/ViewModel/NotificationsViewModel.swift

```diff
@@ -0,0 +1,12 @@
+// NotificationsViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Notifications View Model
+final class NotificationsViewModel: ObservableObject {
+    // MARK: - Public Properties
+
+    @Published var updatesIsOn = false
```

Comment:
```suggestion
    @Published var isOnUpdate = false
```
флаг начинается с is
ниже также